src/app/authentication/components/authenticationSelection.scss

```diff
@@ -4,7 +4,7 @@
 **********************************************************/
 .auth-slection-container {
     display: flex;
-    padding-top: 45px;
+    margin: 45px;
```

Comment:
Just to make sure - did this need to be `margin-top` or was the guidance / fix to make a margin around the entire container?

Reply:
This is not directly related to the accessibility bug, but something I've noticed while working on this page. There is no margin which made the component look weird when the screen is small.